bears/general/KeywordBear.py

```diff
@@ -79,22 +79,55 @@ def run(self,
             filename,
             file,
             keywords: list=['todo', 'fixme'],
+            regex_keyword: str='',
             dependency_results: dict=None):
         '''
         Checks the code files for given keywords.
 
         :param keywords:
             A list of keywords to search for (case insensitive).
             Default are TODO and FIXME.
+        :param regex_keyword:
+            A regular expression to search for matching keywords in a file.
         '''
         keywords_regex = re.compile(
```

Comment:
you gotta change this to somethin' like `simple_keywords_regex` otherwise keywords regex and regex keywords is super confusing